Fullstacker/Api/Controllers/ItemController.cs

```diff
@@ -22,7 +22,9 @@ public ItemController(DataContext dataContext)
         [Route("get")]
         public IActionResult Get()
         {
-            return Ok();
+            var item = _dataContext.Item.ToList();
```

Comment:
Functionality is fine, but we are potentially retrieving multiple items. Hence better to have a plural variable.